src/components/Icon/Icon.stories.tsx

```diff
@@ -16,28 +16,29 @@ const Sizes = {
 export default {
   title: "Components/Icon",
```

Comment:
I noticed that some of the story files have titles, while others do not. Did you mean to delete the titles? (this applies to multiple files)

```suggestion

```